Cargo.toml

```diff
@@ -72,7 +72,8 @@ nix = { version = "0.29.0", features = ["fs", "mount", "signal", "user"] }
 openssl = "0.10.68"
 once_cell = "1.20.2"
 os-release = "0.1.0"
-ostree-ext = "0.15"
+# We pull this one from git, as the project is no longer published as an external crate.
+ostree-ext = { git = "https://github.com/containers/bootc", rev = "dc7d4cf320ec6d8b2470d1d5f76da66ec60b6810" }
```

Comment:
I was confused how this worked actually without a subpath of some kind, but nope [Cargo just searches for every package](https://github.com/rust-lang/cargo/issues/1462#issuecomment-87002475).